Software/src/LogWriter.cpp

```diff
@@ -24,8 +24,12 @@ LogWriter::~LogWriter()
 
 int LogWriter::initWith(const QString& logsDirPath)
 {
+	#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+	#warning wrap log in QStringEncoder?
```

Comment:
I'd rather not leave this open question around, it's unlikely to be addressed at a future point. Either do it or add a comment that we're not doing it because...